appconfiguration/azure-spring-cloud-feature-management/dynamic-features-sample/pom.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>org.springframework.boot</groupId>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <version>2.6.7</version>
+        <relativePath /> <!-- lookup parent from repository -->
+    </parent>
+    <groupId>com.azure.spring</groupId>
+    <artifactId>dynamic-features-sample</artifactId>
+    <version>0.0.1-SNAPSHOT</version>
+    <name>Dynamic Features</name>
+    <description>Shows how to use Dynamic Features in a Spring application</description>
+    <properties>
+        <java.version>11</java.version>
+    </properties>
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>com.azure.spring</groupId>
+            <artifactId>spring-cloud-azure-feature-management-web</artifactId>
+            <version>4.0.0-beta.1</version>
```

Comment:
We should add this to the `<dependencyManagement>` section in the parent pom file, to follow this repo's convention.

Reply:
Is this still being used? I don't see it anywhere.

Reply:
Check this https://github.com/Azure-Samples/azure-spring-boot-samples/blob/3f71bd7ae2521a1505110289a8835ad3405ebfe9/pom.xml#L80-L127

Reply:
This should be fixed.